src/main.rs

```diff
@@ -71,6 +75,16 @@ async fn main() -> anyhow::Result<()> {
     Ok(())
 }
 
+async fn index(skip: Vec<String>) -> anyhow::Result<()> {
+    let credentials = Credentials::read_credentials().await?;
+
+    // TODO: change in flight limit handling? Hardcoded default value?
+    let sw_client = SwClient::new(credentials, 8).await?;
```

Comment:
Instead of introducing a ToDo for this 😅 : 
I would suggest directly adding a constant under the `SwClient` impl block, so you can write
```suggestion
    let sw_client = SwClient::new(credentials, SwClient::DEFAULT_IN_FLIGHT).await?;
```

And the implementation:
```rust

impl SwClient {

    pub const DEFAULT_IN_FLIGHT: usize = 8;
   // ...
}
```

Reply:
You can then also change it in the `auth` function, see https://github.com/shopware/sw-sync-cli/pull/8/files#diff-42cb6807ad74b3e201c5a7ca98b911c5fa08380e942be6e4ac5807f8377f87fcL82

Reply:
Yeah, you're right :laughing: